templates/BUILD.tpl

```diff
@@ -23,13 +23,13 @@ alias(
 platform(
     name = "platform_linux",
     constraint_values = [
-        "@platforms//cpu:%{platform_local_arch_target}",
+        "@platforms//cpu:%{default_arch}",
```

Comment:
Looks like this will wind up being either `arm64` or `amd64`? `arm64` is valid (it's an alias of `aarch64`) but unfortunately it looks like `amd64` is not defined: https://github.com/bazelbuild/platforms/blob/main/cpu/BUILD

Reply:
Ah I missed that (I don't use the default platform in my setup).

I split up `default_arch` into 2 separate variables accordingly.